Tweak.xm

```diff
@@ -568,6 +624,16 @@ void initFlipSwitch()
 %end
 %end
 
+static BOOL didHook = NO;
+static void notificationCallback(CFNotificationCenterRef center, void *observer, CFStringRef name, const void *object, CFDictionaryRef userInfo) {
+    if ([((__bridge NSDictionary *)userInfo)[NSLoadedClasses] containsObject:@"CCUIConnectivityAirDropViewController"]) { // The Network Bundle is Loaded
+        if (!didHook) {
+            didHook = YES;
+            %init(ControlCenter11);
```

Comment:
Can you remove the notification observer afterwards? Just to clean things up and not leave the notification listener active forever